src/views/checkout/AddressInformationView.vue

```diff
@@ -9,6 +9,16 @@
                 v-model:address="selectedDeliveryAddress"
                 @new-address-saved="triggerForQueryingOfBillingAddresses++"
             />
+            <v-text-field
+                clearable
+                label="VAT Identification Number"
+                prepend-icon="mdi-card-account-details"
+                required
```

Comment:
why is this required? the rule also seems to allow an empty string

Reply:
Copy-paste mistake... my bad. Will remove it.

Reply:
00fb73f28089e2bd0ced3fa889dac139e14d6745